test/http-test.js

```diff
@@ -83,6 +83,23 @@ describe('HTTP', function() {
     await wallet.open();
   });
 
+  it('should enable seed phrase recovery', async () => {
+    const options = {
+      passphrase: 'PASSPHRASE'
+    };
+
+    const testwallet = await wclient.createWallet('testwallet', options);
+    assert.strictEqual(testwallet.master.encrypted, false);
+
+    await wclient.lock('testwallet');
+    const masterLock = await wclient.getMaster('testwallet');
+    assert.strictEqual(masterLock.encrypted, true);
+
+    await wclient.unlock('testwallet', 'PASSPHRASE', 100);
+    const masterUnlock = await wclient.getMaster('testwallet');
+    assert.strictEqual(masterUnlock.encrypted, false);
+  });
```

Comment:
I have reached the end of the test, but I have not seen you **actually recover a seed phrase!!**

Be sure you "prove" with the test that the phrase is unattainable when the wallet is locked (maybe `assert` that `phrase` is `undefined` ?)

You can start the test with a phrase you invent and pass it to `createWallet`. I recommend the scary looking `zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo wrong` or the even more daunting `abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon about` ;-)